### PR TITLE
[MISC] Modify dashboard routes and controllers in BACKEND

### PR DESCRIPTION
**Pull request by afei088**
- added validate user middleware to dashboard routes
- changed path parameter id to symbol
- updated dashboard controller

### CODE REVIEW

**Review by kyzh0 — APPROVED** [review]
lgtm

File: backend/src/routes/dashboard.js [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
+router.get('/time-series/weekly/:symbol', validateUser, DashboardController.getTimeSeriesWeekly);
 
-router.get('/time-series/monthly/:id', DashboardController.getTimeSeriesMonthly);
+router.get('/time-series/monthly/:symbol', validateUser, DashboardController.getTimeSeriesMonthly);
 
-router.get('/trending-stocks', DashboardController.getTrending);
+router.get('/trending-stocks', validateUser, DashboardController.getTrending);
 
-router.get('/predict-price/:id', DashboardController.predictPrice);
+router.get('/predict-price/:symbol', validateUser, DashboardController.predictPrice);
```
Comment (YungiKim1999) [inline backend/src/routes/dashboard.js:20]: Does this validateUser get used anywhere within the methods?

Comment (afei088) [reply backend/src/routes/dashboard.js:20]: yes, take a look at the validateUser function in middleware

**Review by YungiKim1999 — APPROVED** [review]
Amazin work Andrew. LGTM!!

**Pull request merged**